config/routes.rb

```diff
@@ -437,13 +437,13 @@
 
       namespace :statistics do
         get '/' => 'statistics#index'
-        get 'answer/:id' => 'answers#question_answer_details'
+        get 'answer/:id' => 'answers#show'
         get 'assessments' => 'aggregate#all_assessments'
         get 'students' => 'aggregate#all_students'
         get 'staff' => 'aggregate#all_staff'
         get 'course/progression' => 'aggregate#course_progression'
         get 'course/performance' => 'aggregate#course_performance'
-        get 'submission_question/:id' => 'answers#all_answers'
+        get 'submission/:submission_id/question/:question_id' => 'answers#all_answers'
```

Comment:
nit: similar to `answers/:id`, idiomatic pattern should be `submissions/:submission_id/questions/:question_id`